README.md

```diff
@@ -1,14 +1,891 @@
+  - [Sequence Diagram ISO Messages](#sequence-diagram-iso-messages)
+  - [Activity Diagram](#activity-diagram)
+    - [Cache Interactions](#cache-interactions)
+      - [Pain013](#pain013)
+  - [Repository](#repository)
+  - [Pain001 Message](#pain001-message)
+  - [Pain013 Message](#pain013-message)
+  - [Pacs002 Message](#pacs002-message)
+  - [Pacs008 Message](#pacs008-message)
+
+## Sequence Diagram ISO Messages
+
+[https://github.com/frmscoe/uml-diagrams/blob/main/services/Transaction-Monitoring-Service-Sequence.puml](https://github.com/frmscoe/uml-diagrams/blob/main/services/Transaction-Monitoring-Service-Sequence.puml)
+
+![](./images/image-20230815-100606.png)
+
+Transaction monitoring is one of many risk management activities that an organization must perform to ensure healthy operations and also to remain compliant with relevant legislation or regulations.
+
+Currently, it accepts ISO20022 Pain001.001.11 message [https://lextego.atlassian.net/browse/AM-632](https://lextego.atlassian.net/browse/AM-632) , pain.013 message [https://lextego.atlassian.net/browse/AM-529](https://lextego.atlassian.net/browse/AM-529) , pacs.002.001.12 [https://lextego.atlassian.net/browse/AM-373](https://lextego.atlassian.net/browse/AM-373) and pacs.008.001.10 [https://lextego.atlassian.net/browse/AM-679?atlOrigin=eyJpIjoiMTFmNzYyOWZmZTk2NDlkNjhhZmVkODlhYTExMmJlM2IiLCJwIjoiaiJ9](https://lextego.atlassian.net/browse/AM-679?atlOrigin=eyJpIjoiMTFmNzYyOWZmZTk2NDlkNjhhZmVkODlhYTExMmJlM2IiLCJwIjoiaiJ9)
+
+## Activity Diagram
+
+The activity diagram below applies to Pain001, Pain013, Pacs008 and Pacs002 messages.
+
+[https://github.com/frmscoe/uml-diagrams/blob/main/services/Transaction-Monitoring-Service.plantuml](https://github.com/frmscoe/uml-diagrams/blob/main/services/Transaction-Monitoring-Service.plantuml)
+
+![](./images/image-20230815-100624.png)
+
+## High Level Cache Interactions
+
+### Pain001
 ![Pain001](https://github.com/frmscoe/tms-service/blob/assets/assets/pain001.svg)
 
-## Pain013
+### Pain013
 ![Pain013](https://github.com/frmscoe/tms-service/blob/assets/assets/pain013.svg)
 
-## Pain008
+### Pain008
 ![Pacs 008](https://github.com/frmscoe/tms-service/blob/assets/assets/pacs008.svg)
 
-## Pacs002
+### Pacs002
 ![Pacs 002](https://github.com/frmscoe/tms-service/blob/assets/assets/pacs002.svg)
 
+### Cache Interactions
+
+#### Pain013
+
+![](./images/Pain013_DB_High_Level_Interactions.png)
+
+```mermaid
+graph TD
+    A([Start]) --> B{Check Cache}
+    B -->|Cache Hit?| C[Retrieve Pain.001 from Cache]
+    B -->|No| D[Get Pain.001 from Database]
+    D --> E[Cache Pain.001]
+    C --> F[Insert TransactionHistory]
+    E --> F
+    F --> G[Insert Creditor and Debtor Accounts]
+    G --> H[Save Transaction Relationship]
+    H --> I[Send to CRSP]
+    I --> J([End])
+```
+
+## Repository
+
+[GitHub - frmscoe/tms-service](https://github.com/frmscoe/tms-service)
+
+## Pain001 Message
```

Comment:
For long code segments like this, can we perhaps try this approach:

![image](https://github.com/frmscoe/tms-service/assets/123470803/1f5b22c8-1879-4660-aa52-2e7c61bb69d1)

where "summary" is the heading (e.g. pain.001 Message) and "details" is the code/json-block (wrapped in formatting ticks)?

Ref here: https://stackoverflow.com/questions/63023659/how-can-i-have-a-code-block-with-a-vertical-scrolling-feature-in-the-markdown-on